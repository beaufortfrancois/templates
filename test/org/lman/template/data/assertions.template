Foo exists.
{{!foo Foo existence}}
{{?_.errors}}
  Fail!
{{/_.errors}}
Bar does not exist.
{{!bar Bar existence}}
{{_.errors}}
