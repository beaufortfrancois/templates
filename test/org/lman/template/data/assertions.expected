Foo exists.
Bar does not exist.
Failed to resolve 'bar' at line 7 column 4 in &lt;root&gt; (Bar existence)
